tokio-trace/Cargo.toml

```diff
@@ -30,6 +30,7 @@ ansi_term = "0.11"
 humantime = "1.1.1"
 futures = "0.1"
 log = "0.4"
+criterion = { version = "0.2", default_features = false }
```

Comment:
What default features are being disabled? It would be nice to have a comment here stating what we're opting out of.